src/plugins/cosmos/components/modals/Staking/views/Overview.tsx

```diff
@@ -24,24 +22,34 @@ export const Overview = ({ assetId }: StakedProps) => {
   }))(assetId) as Asset
   return (
     <AnimatePresence exitBeforeEnter initial={false}>
-      <Box pt='38px' pb='70px' px='34px'>
-        <ModalCloseButton borderRadius='full' />
+      <Box p='22px'>
```

Comment:
`<Overview />` is missing slide transition. Is it possible to have it when going from Stake/Unstake to Overview, but not on the initial render?

Reply:
Seems like this is not low effort, it would require some work to make this possible because every time we load Overview it's a new render

Reply:
Lets do a follow up PR for this, so we can engineer this thing properly

Reply:
Agreed in Discord that the solution to have the `<SlideTransition />` when going back to Overview, but not on initial mount, will need further engineering